src/App.vue

```diff
@@ -1,39 +1,28 @@
 <template>
-  <div>
-    <nav>
-      <router-link to="/">Home</router-link> |
-      <router-link to="/about">About</router-link>
-    </nav>
+  <DefaultLayout>
```

Comment:
```suggestion
  <default-layout>
```
Mình dùng 1 chuẩn thôi em nha. Bên kia anh thấy để `router-view` rồi